x-pack/plugin/security/cli/src/test/java/org/elasticsearch/xpack/security/cli/HttpCertificateCommandTests.java

```diff
@@ -699,11 +709,11 @@ private void verifyKibanaDirectory(Path zipRoot, boolean expectCAFile, Iterable<
         assertThat(kibanaReadme, containsString("https://"));
         assertThat(kibanaReadme, containsString("elasticsearch-ca.pem"));
         readmeShouldContain.forEach(s -> assertThat(kibanaReadme, containsString(s)));
-        shouldNotContain.forEach(s -> assertThat(kibanaReadme, not(containsString(s))));
+        shouldNotContain.forEach(s -> runForNonEmptyPattern(s, () -> assertThat(kibanaReadme, not(containsString(s)))));
 
         assertThat(kibanaYml, containsString("elasticsearch.ssl.certificateAuthorities: [ \"config/elasticsearch-ca.pem\" ]"));
         assertThat(kibanaYml, containsString("https://"));
-        shouldNotContain.forEach(s -> assertThat(kibanaYml, not(containsString(s))));
+        shouldNotContain.forEach(s -> runForNonEmptyPattern(s, () -> assertThat(kibanaYml, not(containsString(s)))));
     }
```

Comment:
I would've not included the empty password in the list of `shouldNotContain`, but it's just a personal preference.

Reply:
Makes sense. The list is not filtered before passed in.